### PR TITLE
[12.0][ADD] hr_employee_ssn: expose SSN & SIN fields

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
related to https://github.com/odoo/odoo/pull/28174

File: hr_employee_ssn/README.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_ssn
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/116/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows using employee's SSN & SIN fields by exposing them in the user interface.
```
Comment (pedrobaeza) [inline hr_employee_ssn/README.rst]: s/using/to use

File: hr_employee_ssn/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    'category': 'Human Resources',
+    'website': 'https://github.com/OCA/hr',
+    'author':
+        'Braibean Apps, '
+        'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'application': False,
+    'summary': (
+        'Allows using employee\'s SSN & SIN fields by exposing them in the'
```
Comment (pedrobaeza) [inline hr_employee_ssn/__manifest__.py]: Better a shorter explanation: `"Show employee's SSN & SIN"

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: hr_employee_ssn/README.rst (hunk 1 above)
Comment (alexey-pelykh) [reply hr_employee_ssn/README.rst]: Done

File: hr_employee_ssn/__manifest__.py (hunk 2 above)
Comment (alexey-pelykh) [reply hr_employee_ssn/__manifest__.py]: Done

**Review by pedrobaeza — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
@luistorresm I'd like to request your review as well

File: hr_employee_ssn/views/hr_employee_views.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <!--
+      Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+      License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+    -->
+
+    <record id="view_employee_form" model="ir.ui.view">
+        <field name="name">hr.employee.form.inherit</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="groups_id" eval="[(6, 0, [ref('hr.group_hr_user') ])]"/>
```
Comment (luistorresm) [inline hr_employee_ssn/views/hr_employee_views.xml]: Hello @alexey-pelykh 

The [page](https://github.com/odoo/odoo/blob/12.0/addons/hr/views/hr_views.xml#L80) where you add this fields already have this group,

Could you remove this?

Regards

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**